zeekpkg/manager.py

```diff
@@ -920,17 +931,44 @@ def _refresh_source(self, name, aggregate=False, push=False):
                     parser.set(qualified_name, 'url', url)
                     parser.set(qualified_name, 'version', version)
 
+                    if qualified_name not in prev_packages:
```

Comment:
The `qualified_name` has the `index_dir` prepended, does that match what's in the `prev_packages` set?

Reply:
Yes, I think so. The `index_dir` is the `corelight` in something like `corelight/zeek-quic`, computed from the local paths on disk. The logic that the previous code uses to populate the sections in the config parser hasn't changed, and `prev_packages` is just a set of those section titles. It's definitely tricky, though ... perhaps we should rename `prev_packages` to `prev_qualified_names`?

Reply:
Got it, it was the naming of `index_dir` that threw me off. The way it's named and computed didn't let me think we'd arrive at just that namespace label.

All good then, ready to merge from my end.